Extension/src/common.ts

```diff
@@ -775,7 +776,7 @@ export async function spawnChildProcess(program: string, args: string[] = [], co
         } else {
             stdout = localize('process.succeeded', 'Process executed successfully.');
         }
-        return { succeeded: true, exitCode, output: stdout };
+        return { succeeded: true, exitCode, outputError: programOutput.stderr, output: stdout };
```

Comment:
It seems a little odd for the outputError named parameter to not be the last one since it's the last declared, but I'm not sure if that matters or not.

Reply:
The order shouldn't matter.